File: sub-web-model/src/main/java/org/github/base/mapper/ISysRoleMenuMapper.java
package org.github.base.mapper;

import org.github.base.entity.SysRoleMenuEntity;
import org.github.base.IMapper;
import org.github.mybatis.MyBatisMapper;

/**
 * <p>
 * 角色与菜单对应关系 Mapper 接口
 * </p>
 *
 * @author JYD_XL
 * @since 2021-05-07
 */
@MyBatisMapper
public interface ISysRoleMenuMapper extends IMapper<SysRoleMenuEntity> {}

File: sub-mysql-mydb-service/src/main/java/org/github/mysql/mydb/base/mapper/ISysRoleMenuMapper.kt
@file:Suppress("RedundantSemicolon")

package org.github.mysql.mydb.base.mapper;

import org.github.mysql.mydb.base.entity.SysRoleMenuEntity;
import org.github.base.IMapper;
import org.github.mybatis.MyBatisMapper;

/**
 * 角色-菜单关系表 Mapper 接口
 *
 * @author JYD_XL
 */
@MyBatisMapper
interface ISysRoleMenuMapper : IMapper<SysRoleMenuEntity>
